tutorials/scripting/c_sharp/c_sharp_variant.rst

```diff
@@ -0,0 +1,156 @@
+.. _doc_c_sharp_variant:
+
+C# Variant
+==========
+
+For a detailed explanation of Variant in general, see the :ref:`Variant <class_Variant>` documentation page.
+
+``Godot.Variant`` is used to represent Godot's native :ref:`Variant <doc_variant_class>` type. Any Variant-compatible type can be converted from/to it.
```

Comment:
And this as well.

Reply:
I'll remove the word "native" then, right?

Reply:
I think "native" here makes sense as in "native to the engine", but if you think it's confusing in any way, feel free to drop it.

Reply:
Ah, I can see that. It's probably less confusing **with** "native".